### ISSUE TITLE
Dependencies out of date

### ISSUE THREAD

**Issue by nniehoff** [nautobot/nautobot-plugin-chatops#41]
### Environment
* Python version:  3.6.10
* Nautobot version:  1.0.1
* nautobot-plugin-chatops version:  1.2.0

When adding "social-auth-core[openidconnect]" per the Nautobot documentation to enable OIDC SSO authentication I ran into a dependency problem showing this plugin needs to have it's dependencies updated:

```no-highlight
poetry add 'social-auth-core[openidconnect]'
Using version ^4.1.0 for social-auth-core
Updating dependencies
Resolving dependencies... (1.0s)
  SolverProblemError
  Because no versions of social-auth-core match >4.1.0,<5.0.0
   and social-auth-core (4.1.0) depends on PyJWT (>=2.0.0), social-auth-core (>=4.1.0,<5.0.0) requires PyJWT (>=2.0.0).
  Because pyjwt (1.7.1) depends on pyjwt (1.7.1)
   and no versions of pyjwt match >1.7.1,<2.0.0, pyjwt (>=1.7.1,<2.0.0) requires pyjwt (1.7.1).
  Thus, social-auth-core (>=4.1.0,<5.0.0) is incompatible with pyjwt (>=1.7.1,<2.0.0).
  And because nautobot-chatops (1.2.0) depends on pyjwt (>=1.7.1,<2.0.0)
   and no versions of nautobot-chatops match >1.2.0,<2.0.0, social-auth-core (>=4.1.0,<5.0.0) is incompatible with nautobot-chatops (>=1.2.0,<2.0.0).
  So, because nautobot-project depends on both nautobot-chatops (^1.2.0) and social-auth-core (^4.1.0), version solving failed.
  at ~/.poetry/lib/poetry/puzzle/solver.py:241 in _solve
      237│             packages = result.packages
      238│         except OverrideNeeded as e:
      239│             return self.solve_in_compatibility_mode(e.overrides, use_latest=use_latest)
      240│         except SolveFailure as e:
    → 241│             raise SolverProblemError(e)
      242│ 
      243│         results = dict(
      244│             depth_first_search(
      245│                 PackageNode(self._package, packages), aggregate_package_nodes
```

### Steps to Reproduce
1.  Add both the chatops plugin and social auth opened packages

### Expected Behavior
SSO Auth should work as documented

### Observed Behavior
Unable to install SSO and chatops plugin

**Comment by jvanderaa** [comment]
@matt852 Would you be able to take this one up? The pyjwt package appears to only be used by MS Teams. I am thinking that we would be able to make the update, there were no breaking changes called out on the rev to version 2, but a significant number of changes in general.

**Comment by matt852** [comment]
@jvanderaa Sure

**Comment by jvanderaa** [comment]
Closed from #44.
